src/feedback_forensics/app/data/loader.py

```diff
@@ -237,7 +246,8 @@ def get_votes_dict_from_annotated_pairs_json(results_path: pathlib.Path) -> dict
         else:
             annotator_metadata[annotator_id] = {
                 "variant": "nondefault_annotator",
-                "annotator_visible_name": annotator_info.get("name", annotator_id),
+                "annotator_visible_name": PREFIX_OTHER_ANNOTATOR
+                + annotator_info.get("name", annotator_id),
```

Comment:
why don't we use the variant/type here as well?

Reply:
Good point. We previously dropped this info when loading, by setting `"variant": "nondefault_annotator",
`, so I didn't add it to the visible name either. Do you know if there's a reason for that, or should we use `"variant": annotator_info.get("type", "unknown")` instead? Or would you say that the concept of "variant" should differ from type?

Reply:
Oh I see, I think this just for legacy reasons - the variant info inside FF predates the annotated pairs format if I remember correctly. Let's just do it as you suggest and add the type from annotated pairs as the variant for non-principle and non-default annotators.

Reply:
Done, also dropped the "Other" from the prefix.